lib/sinon/util/core/deprecated.js

```diff
@@ -0,0 +1,24 @@
+/*eslint no-console: 0 */
+"use strict";
+
+// wrap returns a function that will invoke the supplied function and print a deprecation warning to the console each
+// time it is called.
+exports.wrap = function (func, msg) {
+    return function () {
+        // Watch out for IE7 and below! :(
+        if (typeof console !== "undefined") {
+            if (console.warn) {
```

Comment:
Not seeing much trace of the `console.info` instead of `console.warn` thing you talked about for PhantomJS?